qchem/pennylane_qchem/qchem/structure.py

```diff
@@ -706,8 +706,9 @@ def convert_observable(qubit_observable, wires=None):
       0.04475014 -0.04475014 -0.04475014  0.04475014  0.12293305  0.16768319
       0.16768319  0.12293305  0.17627641]
     """
-
-    if any(np.iscomplex(np.array(coef)) for coef in qubit_observable.terms.values()):
+    if any(
+        np.iscomplex(np.real_if_close(coef, tol=1e08)) for coef in qubit_observable.terms.values()
```

Comment:
@agran2018 it might be a good idea to include `tol` as a keyword for the function, so that end users can modify the tolerance if they want:

```python
def convert_observable(qubit_observable, wires=None, tol=1e8):
    if any(
        np.iscomplex(np.real_if_close(coef, tol=tol)) for coef in qubit_observable.terms.values()
    ):
```